docs/it/università-vs-self-made.md

```diff
@@ -0,0 +1,44 @@
+
+**_“Come posso trovare gli ITS?”_**
+
+Prima di rispondere a questa domanda, è bene precisare che gli ITS in Italia offrono una vasta gamma di corsi dedicati a settori diversi, tra cui tecnologia, industria, e servizi. Per ovvi motivi in questo libro, più precisamente in questo capitolo, si parlerà di quelli legati al settore IT.
+
+Chiusa questa parentesi, il sito dell’**_[INDIRE](https://www.indire.it/progetto/its-istituti-tecnici-superiori/dove-sono-gli-its/)_** (_Istituto Nazionale Documentazione Innovazione Ricerca Educativa_) è sicuramente la migliore risorsa per ottenere una risposta a questa domanda. Ovviamente non l’unica, ma sicuramente tra tutte le pagine risulta essere la più completa, presentando anche a sostegno i dati relativi agli ITS della regione e all’occupazione degli studenti diplomati.
+
+**“_Perché l’ITS potrebbe risultare un’ottima via?”_**
+
+Innanzitutto come già citato è un corso molto breve (2 anni), a numero chiuso (quindi con un test di ammissione) e alla portata di tutti dal punto di vista economico, dal momento che l’iscrizione è gratuita o a costo molto ridotto. Solitamente viene fornito un PC da poter utilizzare durante le lezioni e dedicato completamente alla programmazione, ma soprattutto, ciò che contraddistingue l’ITS (e il motivo per cui viene citato in questo capitolo) è la sua **dualità tra lezioni in aula e stage in azienda** per poter testare in modo pratico “sul campo” i concetti appresi.
+
+È vero, la preparazione concettuale e teorica potrebbe risultare meno solida rispetto a quella universitaria, ma è sicuramente più vantaggiosa in ambito aziendale, essendo maggiormente orientata al mondo del lavoro. Proprio in questo contesto, sfruttando le opportunità lavorative e gli incontri con i professionisti del settore, dobbiamo far emergere la nostra proattività e curiosità per valorizzare al meglio le competenze di base che abbiamo acquisito!
+
+Per esempio, le lezioni di Python che ho seguito in Accademia hanno solleticato la mia curiosità e adesso non vedo l'ora di affinare le mie conoscenze su questo linguaggio. Partendo da questa "infarinatura" posso apprendere più facilmente, anche in maniera autonoma.
+
+Tramite questo percorso, passando per gli stage, si possono acquisire anche competenze su argomenti che non sono stati affrontati in aula. Ad esempio, in futuro potrei trovarmi a svolgere uno stage in un’azienda che usa un framework specifico: avendo seguito le lezioni su JavaScript, ho acquisito delle solide basi che non potranno far altro che semplificare il processo di apprendimento di una nuova tecnologia basta su questo linguaggio di programmazione.
```

Comment:
Come sopra. Rimuoverei la parte soggettiva, e illustrerei i vantaggi da un punto di vista più fattuale (perché, oggettivamente, ci sono!)

```suggestion
Attraverso gli stage che si svolgono seguendo questo percorso, si possono acquisire anche competenze su argomenti che non sono stati affrontati in aula. Ad esempio, in futuro ...```

Reply:
Anche in questo caso la motivazione per cui ho scelto questa forma è quella menzionata nella review precedente. Ovviamente se modifico uno di questi due paragrafi per renderli più "oggettivi", andrò ad agire anche sull'altro per tenerli coerenti